.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: scala
+
+sudo: required
+
+scala:
+   - 2.11.8
+
+jdk:
+   - oraclejdk8
+
+services:
+  - docker
+
+before_install:
+  - docker-compose up -d
+
+script:
+  - sbt clean coverage test coverageReport coverageAggregate
+
+before_cache:
+  - find $HOME/.ivy2 -name "ivydata-*.properties" -print -delete
+  - find $HOME/.sbt  -name "*.lock"               -print -delete
+
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot
+
+#deploy:
+#  provider: script
+#  script:
+#    - sbt publish
+#  on:
+#    tags: true
+#    repo: firstbirdtech/akka-persistent-scheduler
+
+env:
+  global:
+    # encrypt with: travis encrypt BINTRAY_USER=...
+    - secure: "WXITvMAQmjJHQijqCkS+cZ9BLEH+B9n3IiGtP3bV8U+9UsR4kU7+mZyDAQDdI6l36rHDgPPWqy/PCxnXETKY0do2RqI70lk3bgqmPUy7V6BP+pweX103fBuztEuPBm124nnUrJj7enRGDe+kwSoXCCwkidw3RjrJy2yNJBLgOSu7HPDfOFqnzsHiA/C1aithTCG0V7e0l6Lvb8HzXRg91pTAJGJQIm3Mff4mACpThyL90hfde+Jb2Is1KnpXTMCoGWC65wxbh8rjCmPSFs+98XeceAFDJqXiVKx/AWJDVMv11NYxOGqBDjC2w6LgBSvI140xm5Y7LhgeM5nZNiUaximoxAbmHFrddVOyxZ9jlNe+OLxo3FQXaa6+6MtXifZDbL3ikz/Ztyt0AE79o1Ui/GFLs2pKc2g1WlkelMH5qLmXt00lKzn1f/NN3k18WaBhQlKPosrEH0mKWlfyrkPETHi7e2fCh9UrKhDZw3/SjdFGMLMZfQr3JJCSBxusfngBMVQZ1ZJ3Fpzn2HteL1XLrssPUk+S9s+fNALW0UuaUu7rnEyjFCSnDoraTpjQboZXbpWz5qI/pIa5xcSv53jblYlwvRJbgsjc2LO1k/mHrZfJ8jVoo9vL04V0ch8kCCATeqaocdg2Wlb3fj0j5/hVwrDYJffoVlaHo2lB99yO+ds="
```

Comment:
Do I need to? Why not use the same user, if we will all be using the same creds?